### ISSUE TITLE
All: "Elevation" on Zeplin shows "flat". Github site doesn't.

### ISSUE THREAD

**Issue by McCallumDillon** [JDRF/design-system#13]
the "flat" state is sort of assumed, but we do have it shown on Zeplin and it isn't reflected on the github site.

![screen shot 2016-05-27 at 1 42 12 pm](https://cloud.githubusercontent.com/assets/5239235/15620783/42f76110-2411-11e6-9e4b-7f7844dec4de.jpg) <-- zeplin: far left element is the "flat" state.

**Comment by RachelRVasquez** [comment]
For the flat state, visually in our DS where the background color is white, it didn't make sense to have a white square without any shadow. If we added it, we'd have to brainstorm how to visually display it so it's something we can see against the white background unless we changed the design of the elevations section. There was a similar issue in Typography to show white text colors against white and I included a note about the colors instead with the color values. Perhaps we can do the same here - provide a note about the flat state and where it's applied? @McCallumDillon @fuhton
